test/multiple-set-get.test.ts

```diff
@@ -30,7 +30,11 @@ describe('multiple get and set', () => {
 
     // Get multiple keys
     const getResp = await client.mget(key1, key2);
-    expect(getResp).toEqual([value, null]);
+    if (process.env.MOMENTO_ENABLED === 'true') {
+      expect(getResp).toEqual([value]);
+    } else {
+      expect(getResp).toEqual([value, null]);
```

Comment:
this feels wrong to me. I think we should be able to return the same result that redis does here. Can you help me understand why we can't?

Reply:
Because GetBatch Response from sdk returns a map/record of keys and values for only the keys that exist.

Reply:
https://github.com/momentohq/client-sdk-javascript/blob/cbd07069bc90d960d02549952fe8a9f0cd1b4adb/packages/core/src/messages/responses/cache-batch-get.ts#L85

Reply:
it also exposes a `results` function to get the raw results. But even with the map/record versions, you have the list of keys that they asked for, in the order that they asked for them, so you could just add `null`s into the return value for anything that didn't show up in the map, right?